src/lightning/pytorch/__init__.py

```diff
@@ -23,6 +23,7 @@
     _logger.propagate = False
 
 from lightning.fabric.utilities.seed import seed_everything  # noqa: E402
+from lightning.pytorch import demos  # noqa: E402
```

Comment:
See https://github.com/Lightning-AI/lightning/pull/18451#discussion_r1319560055

Either we do this or change the import in the example. cc @Borda

Reply:
I strongly suggest that this isn't imported by default. It's an unnecessary overhead for 99.99% of users.